app/src/main/java/org/oppia/app/home/recentlyplayed/OngoingListAdapter.kt

```diff
@@ -61,34 +61,46 @@ class OngoingListAdapter(
         storyGridPosition = position - titleIndex
```

Comment:
The logic changed in this file is incorrect because earlier using `%2` was correct because there were 2 items but here each row can have 3 items and therefore `%3` should be used.

For reference check `TopicListAdapter`

Reply:
Done.